packages/botonic-react/src/components/message.jsx

```diff
@@ -8,12 +8,14 @@ import { Button } from './button'
 import { Reply } from './reply'
 import { WEBCHAT, COLORS } from '../constants'
 import Fade from 'react-reveal/Fade'
+import moment from 'moment'
 
 const MessageContainer = styled.div`
   display: flex;
   justify-content: ${props => (props.isfromuser ? 'flex-end' : 'flex-start')};
   position: relative;
   padding-left: 5px;
+  padding-right: ${props => props.userRightPadding};
```

Comment:
maybe it would be better to be able to redefine the container, not only the right padding